spec/features/state_file/nj/complete_intake_spec.rb

```diff
@@ -0,0 +1,392 @@
+      expect(page).to have_text I18n.t("state_file.questions.nj_homeowner_eligibility.edit.title", filing_year: filing_year)
+      checkboxes.each do |checkbox|
+        expect(page).to have_field(I18n.t("state_file.questions.nj_homeowner_eligibility.edit.#{checkbox}"), checked: expected_state)
+      end
+    end
+
+    def expect_all_tenant_checkboxes_state(checkboxes, expected_state)
+      expect(page).to have_text I18n.t("state_file.questions.nj_tenant_eligibility.edit.title", filing_year: filing_year)
+      checkboxes.each do |checkbox|
+        expect(page).to have_field(I18n.t("state_file.questions.nj_tenant_eligibility.edit.#{checkbox}"), checked: expected_state)
+      end
+    end
+
+    def fill_property_tax_paid(amount, municipality = "Atlantic City")
+      expect(page).to have_text I18n.t("state_file.questions.nj_homeowner_property_tax.edit.title", filing_year: filing_year, municipality: municipality)
+      fill_in I18n.t('state_file.questions.nj_homeowner_property_tax.edit.label', filing_year: filing_year), with: amount
+      continue
+    end
+
+    def fill_rent_paid(amount)
+      expect(page).to have_text I18n.t("state_file.questions.nj_tenant_rent_paid.edit.title", filing_year: filing_year)
+      fill_in I18n.t('state_file.questions.nj_tenant_rent_paid.edit.label', filing_year: filing_year), with: amount
+      continue
+    end
+
+    def expect_estimated_tax_page
+      expect(page).to have_text I18n.t("state_file.questions.nj_estimated_tax_payments.edit.title", filing_year: filing_year)
+    end
+
+    def expect_ineligible_page(property, reason)
+      valid_property_values = ["on_home", "on_rental", nil]
+      valid_reasons = ["multi_unit_conditions", "property_taxes", "neither"]
+      throw "not a valid property value: #{property}" unless valid_property_values.include?(property)
+      throw "not a valid reason value: #{reason}" unless valid_reasons.include?(reason)
+
+      reason_text = I18n.t("state_file.questions.nj_ineligible_property_tax.edit.reason_#{reason}", filing_year: MultiTenantService.statefile.current_tax_year)
+      property_text = property.present? ? I18n.t("state_file.questions.nj_ineligible_property_tax.edit.#{property}") : nil
+      expect(page).to have_text strip_html_tags(I18n.t("state_file.questions.nj_ineligible_property_tax.edit.title_html", property: property_text)).strip
+      expect(page).to have_text strip_html_tags(I18n.t("state_file.questions.nj_ineligible_property_tax.edit.reason_html", reason: reason_text))
+      continue
+    end
+
+    def advance_to_property_tax_page(df_persona_name)
+      advance_to_start_of_intake(df_persona_name)
+      advance_county_and_municipality
+      if has_text? I18n.t("state_file.questions.nj_disabled_exemption.edit.title")
+        advance_disabled_exemption
+      end
+      advance_veterans_exemption
+      if has_text? I18n.t("state_file.questions.nj_college_dependents_exemption.edit.title")
```

Comment:
Similarly, only personas with dependents are shown this controller, so made this conditional as well